packages/carbon-web-components/src/components/number-input/stories/helpers.ts

```diff
@@ -1,44 +1,48 @@
 /**
  * @license
  *
- * Copyright IBM Corp. 2019, 2022
+ * Copyright IBM Corp. 2019, 2023
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 import { action } from '@storybook/addon-actions';
 import { INPUT_SIZE } from '../../input/input';
-import { NUMBER_INPUT_COLOR_SCHEME } from '../number-input';
-
-const colorSchemes = {
-  Regular: null,
-  [`Light (${NUMBER_INPUT_COLOR_SCHEME.LIGHT})`]:
-    NUMBER_INPUT_COLOR_SCHEME.LIGHT,
-};
 
 const sizes = {
-  Regular: null,
   [`Small size (${INPUT_SIZE.SMALL})`]: INPUT_SIZE.SMALL,
-  [`Extra large size (${INPUT_SIZE.EXTRA_LARGE})`]: INPUT_SIZE.EXTRA_LARGE,
+  [`Medium size (${INPUT_SIZE.MEDIUM})`]: INPUT_SIZE.MEDIUM,
+  [`Large size (${INPUT_SIZE.LARGE})`]: INPUT_SIZE.LARGE,
 };
 
-const createProps = ({ boolean, textNullable, number, select }) => ({
-  colorScheme: select('Color scheme (color-scheme)', colorSchemes, null),
+const createProps = ({ boolean, number, select, text }) => ({
+  allowEmpty: boolean('Allow empty (allow-empty)', false),
+  decrementButtonDescription: text(
+    'Decrement button assistive description (decrement-button-assistive-text)',
+    'decrease number input'
+  ),
+  incrementButtonDescription: text(
+    'Increment button assistive description (increment-button-assistive-text)',
+    'increase number input'
+  ),
   disabled: boolean('Disabled (disabled)', false),
-  value: number('Input value (value)', 0),
+  helperText: text('Helper text (helper-text)', 'Optional helper text'),
+  hideLabel: boolean('Hide label (hide-label)', false),
+  hideSteppers: boolean('Hide steppers (hide-steppers)', false),
+  invalid: boolean('Invalid (invalid)', false),
+  invalidText: text('Invalid text (invalid-text)', 'Number is not valid'),
+  label: text('Label (label)', 'number-input label'),
+  readonly: boolean('Read only (readonly)', false),
+  value: number('Value (value)', 50),
+  warn: boolean('Warn (warn)', false),
+  warnText: text('Warn text (warn-text)', 'Warning text'),
   min: number('Minimum value (min)', 0),
   max: number('Maximum value (max)', 100),
   step: number('Value to step the input by (step)', 1),
-  size: select('Input size (size)', sizes, INPUT_SIZE.REGULAR),
-  placeholder: textNullable(
-    'Placeholder text (placeholder)',
-    'Optional placeholder text'
-  ),
-  invalid: boolean('Invalid (invalid)', false),
+  size: select('Size (size)', sizes, INPUT_SIZE.MEDIUM),
   onInput: action('input'),
   mobile: boolean('Mobile mode (mobile)', false),
```

Comment:
is this still necessary for web components? https://github.com/carbon-design-system/carbon/blob/9d0a08fc5ccaf7259f256545b172cbf4dee8991a/docs/migration/v11.md?plain=1#L929

Reply:
oh good catch - looks like it doesn't even have mobile specific styles anymore - will remove!